internal/store/pod.go

```diff
@@ -1708,6 +1709,27 @@ func createPodServiceAccountFamilyGenerator() generator.FamilyGenerator {
 	)
 }
 
+func createPodSchedulerNameFamilyGenerator() generator.FamilyGenerator {
+	return *generator.NewFamilyGeneratorWithStability(
+		"kube_pod_scheduler_name",
+		"The scheduler name for a pod.",
+		metric.Gauge,
+		basemetrics.ALPHA,
+		"",
+		wrapPodFunc(func(p *v1.Pod) *metric.Family {
+			m := metric.Metric{
+				LabelKeys:   []string{"scheduler_name"},
```

Comment:
I am wondering if having something like `kube_pod_scheduler{name="foo"}` could be better to remove the redundancy with the metric name its label.

Reply:
Think that makes sense.  Updated.